joomla/joomla-cms
# Pass lft, level and path to rebuild function.

Pull Request for Issue #19788  .

### Summary of Changes
The ComponentAdapter tries to rebuild the nested table after it deletes a duplicate menuitem.
This PR adds the needed arguments to that rebuild call so the function doesn't mess up the table.


### Testing Instructions
* Install and uninstall any extension and make sure both works and the menuitems are created as expected. No changes to current behavior expected here.
* Install the test component posted in https://github.com/joomla/joomla-cms/issues/19788 and uninstall it again (follow test instructions there), It should work now.

### Expected result
Everything works


### Actual result
The test component produced a messed up menu table.


### Documentation Changes Required
None
